backend/otr/src/main/java/fi/oph/otr/onr/model/PersonalData.java

```diff
@@ -0,0 +1,18 @@
+package fi.oph.otr.onr.model;
+
+import javax.validation.constraints.NotBlank;
+import lombok.Builder;
+import lombok.NonNull;
+
+@Builder
+public record PersonalData(
+  @NonNull @NotBlank String firstName,
```

Comment:
Olisko tänne mielekästä lisätä `onrId`, jotta listan näitä instansseja voisi streamien kautta mäpätä onrId:n perusteella? Muuta käyttöä tuolla kentällä täällä ei olisi.